### PR TITLE
[dom-gpu]  NVIDIA Nsight Compute

### PR DESCRIPTION
**Pull request by jgphpc**
Will fix MAINT-183

### CODE REVIEW

**Comment by jgphpc** [comment]
@jenkins-cscs retest this please

File: easybuild/easyconfigs/n/nvidia-nsight-compute/nvidia-nsight-compute-2019.4.0.12.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+name = 'nvidia-nsight-compute'
+version = '2019.4.0.12'
+homepage = 'https://developer.nvidia.com/nsight-compute-2019_4'
+description = """NVIDIA Nsight Compute is an interactive kernel profiler for
+CUDA applications."""
+
+toolchain = SYSTEM
+sources = ['/apps/common/UES/easybuild/sources/%(nameletter)s/%(name)s/%(name)s-linux-%(version)s.run']
+checksums = ['157687c72397b0dc6f12848e7e10d120']
+install_cmd = sources[0] + " --nox11 --nochown --target %(installdir)s -- -targetpath=%(installdir)s -noprompt "
```
Comment (kraushm) [inline easybuild/easyconfigs/n/nvidia-nsight-compute/nvidia-nsight-compute-2019.4.0.12.eb]: Why is `--accept` not needed anymore?

Comment (jgphpc) [reply easybuild/easyconfigs/n/nvidia-nsight-compute/nvidia-nsight-compute-2019.4.0.12.eb]: it's a different tool. It does not has `--accept`

File: easybuild/easyconfigs/n/nvidia-nsight-compute/nvidia-nsight-compute-2019.4.0.12.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# install_cmd = sources[0] + " --accept --nox11 --nochown --target %(installdir)s -- -targetpath=%(installdir)s -noprompt "
```
Comment (kraushm) [inline easybuild/easyconfigs/n/nvidia-nsight-compute/nvidia-nsight-compute-2019.4.0.12.eb]:
```suggestion
```

**Pull request merged**